sinch/domains/voice/models/svaml/actions.py

```diff
@@ -0,0 +1,114 @@
+from typing import Optional, List, TypedDict
+from sinch.core.models.base_model import SinchRequestBaseModel
+from sinch.domains.voice import Destination, ConferenceDTMFOptions
+
+
+class AnsweringMachineDetection(TypedDict):
+    enabled: bool
+
+
+class CallHeaders(TypedDict):
+    key: str
+    value: str
+
+
+@dataclass
+class HangupAction(SinchRequestBaseModel):
+    name: str = "hangup"
+
+
+@dataclass
+class ContinueAction(SinchRequestBaseModel):
+    name: str = "continue"
+
+
+@dataclass
+class ConnectPtsnAction(SinchRequestBaseModel):
+    name: str = "connectPstn"
+    number: Optional[str] = None
+    locale: Optional[str] = None
+    max_duration: Optional[int] = None
+    dial_timeout: Optional[int] = None
+    cli: Optional[str] = None
+    suppress_callbacks: Optional[bool] = None
+    dtmf: Optional[str] = None
+    indications: Optional[str] = None
+    amd: Optional[AnsweringMachineDetection] = None
+
+    def as_dict(self):
+        payload = super().as_dict()
+        if payload.get("max_duration"):
+            payload["maxDuration"] = payload.pop("max_duration")
+
+        if payload.get("dial_timeout"):
+            payload["dialTimeout"] = payload.pop("dial_timeout")
+
+        if payload.get("suppress_callbacks"):
+            payload["suppressCallbacks"] = payload.pop("suppress_callbacks")
+
+        return payload
+
+
+@dataclass
+class ConnectMxpAction(SinchRequestBaseModel):
+    name: str = "connectMxp"
+    destination: Optional[Destination] = None
+    call_headers: Optional[List[CallHeaders]] = None
+
+    def as_dict(self):
+        payload = super().as_dict()
+        if payload.get("call_headers"):
+            payload["callHeaders"] = payload.pop("call_headers")
+
+        return payload
+
+
+@dataclass
+class ConnectSipAction(SinchRequestBaseModel):
+    destination: Optional[Destination]
+    name: str = "connectSip"
+    max_duration: Optional[int] = None
+    cli: Optional[str] = None
+    transport: Optional[str] = None
+    suppressCallbacks: Optional[bool] = None
+    call_headers: Optional[List[CallHeaders]] = None
+    moh: Optional[str] = None
+
+    def as_dict(self):
+        payload = super().as_dict()
+        if payload.get("max_duration"):
+            payload["maxDuration"] = payload.pop("max_duration")
+
+        if payload.get("suppress_callbacks"):
+            payload["suppressCallbacks"] = payload.pop("suppress_callbacks")
+
+        return payload
+
+
+@dataclass
+class ConnectConfAction(SinchRequestBaseModel):
+    destination: Optional[Destination]
+    conference_id: str
+    name: str = "connectConf"
+    conference_dtmf_options: Optional[ConferenceDTMFOptions] = None
+    moh: Optional[str] = None
+
+
+@dataclass
+class RunMenuAction(SinchRequestBaseModel):
+    name: str = "runMenu"
+    barge: Optional[bool] = None
+    locale: Optional[str] = None
+    main_menu: Optional[str] = None
+    enable_voice: Optional[bool] = None
+    menus: Optional[dict] = None
```

Comment:
`menus` is an array with a complex structure: it needs to be specified

Reply:
already fixed; it was WiP